v2/cmd/voucher_client/root.go

```diff
@@ -53,6 +53,8 @@ func init() {
 	viper.BindPFlag("timeout", rootCmd.Flags().Lookup("timeout"))
 	rootCmd.Flags().StringVarP(&defaultConfig.Check, "check", "c", "all", "the name of the checks to run against Voucher with")
 	viper.BindPFlag("check", rootCmd.Flags().Lookup("check"))
+	rootCmd.Flags().StringVarP(&defaultConfig.Auth, "auth", "a", "basic", "the method to authenticate against Voucher with. Supported types: basic, idtoken")
```

Comment:
Is there a benefit of using a string over a boolean?

Reply:
it's to support other types of auth in the future!